ingestion/src/metadata/ingestion/ometa/utils.py

```diff
@@ -25,8 +25,8 @@
 T = TypeVar("T", bound=BaseModel)  # pylint: disable=invalid-name
 
 
-def ometa_logger():
-    return logging.getLogger("OMetaAPI")
+def ometa_logger(name: str = "OMetaAPI"):
+    return logging.getLogger(name)
```

Comment:
hi! This function was just to help us create loggers with the same name all around the place. If for the `sql_source` we want a different one, instead of passing parameters here I'd rather us have an `Ingestion` logger. In the end it does not matter if the module is named `sql_source` or `snowflake` or whatever, we will already know that depending on the source being run, so this should simplify reading the logs for users.

Thanks!

Reply:
Her @pmbrull, so refering to this issue - https://github.com/open-metadata/OpenMetadata/issues/4267, while on support if we in future try to debug their code / traceback, it will be harder if we have to look at the same named logs.

Reply:
after dicussing: let's format the logs as `logger:file:line` and keep loggers by module. We can have more info in the debug mode. Thanks!